CHANGELOG

```diff
@@ -4,9 +4,7 @@
   * [feature ...]
   * Upgraded gems: capybara, rails, rdoc, rubocop, nokogiri, papertrail, puma,
   * Bugs fixed:
-    - [bug fixed #1]
-    - [bug fixed ...]
-    - Bug tracker items: #N, #M, #O, ...
+    - Fix uploading attachments with newlines
```

Comment:
I'm not sure this describes the issue at hand as it makes it sound it's about the content of the attachment